content/transcripts.fr.md

```diff
@@ -0,0 +1,316 @@
+---
+title: "Transcriptions"
+title_image: /content-images/wai-media-guide/transcript.svg
+nav_title: "Transcriptions"
+
+lang: fr   # change "en" to lang code, here and 2 @@s below
+last_updated: 2020-01-01   # Change to date of translation YYYY-MM-DD (month in middle)
+translators:
+- name: Sofia Ahmed
+contributors: 
+- name: "Sandra Velarde Gonzalez (ETNIC)"
+
+permalink: /media/av/transcripts/fr   # Add lang to end /link/to/page/@@
+ref: /media/av/transcripts/   # Do not change this
+layout: default
+github:
+   repository: w3c/wai-media-guide
+   path: 'content/transcripts.fr.md'   # Add lang to the middle of the filename, e.g., index.@@.md
+
+resource:
+  ref: /media/av/
+navigation:
+  previous: /media/av/captions/
+  next:     /media/av/sign-languages/
+changelog: /media/av/changelog/
+acknowledgements: /media/av/acknowledgements/
+
+description: Vous aide à comprendre et à créer des transcriptions pour rendre les contenus audios et vidéos multimédia accessibles.
+image: /content-images/wai-media-guide/social.png
+
+footer: >   # Translate words below, including "Date:" and "Editor:". (Do not update the date.)
+   <p><strong>Date :</strong> Mise à jour : 9 octobre 2020. JOURNAL DES MODIFICATIONS.</p>
+   <p><strong>Rédactrice :</strong> <a href="http://www.w3.org/People/Shawn">Shawn Lawton Henry</a>. REMERCIEMENTS : liste les contributeurs et les crédits.</p>
```

Comment:
We should not translate "ACKNOWLEDGEMENTS" as it is automatically translated in the process.